android/src/main/java/com/amplitude/android/DefaultTrackingOptions.kt

```diff
@@ -32,15 +32,4 @@ constructor(
                 screenViews = false,
             )
     }
-
-    @ExperimentalAmplitudeFeature
-    constructor(
```

Comment:
Is this backward compatible? 
The original way is not allowed anymore.
```DefaultTrackingOptions(
        sessions = true,
        appLifecycles = true,
        deepLinks = false,
        screenViews = true,
        userInteractions = true
    )
```

Reply:
`userInteractions` has never been released. So I think it's safe to remove it entirely from `DefaultTracking` since no one's using it.